config/api.yml

```diff
@@ -1210,6 +1210,8 @@
         :identifier: ab_group_edit
       - :name: delete
         :identifier: ab_group_delete
+      - :name: order
```

Comment:
I'm thinking the action name should be `reorder`.  To me, `order`, sounds like you're going to purchase something.

Reply:
@Fryguy Do you think `sort` will be a better alternative?